FHIR/sushi
# Fixes SUSHI adding instances of custom resources in the IG + outputs warning

Fixes #1090 

The IG Publisher does not support instances of custom resources. This PR fixes that issue and issues a warning. 

To test:
Test with this FSH code and see if the FormTemplate Instance displays under `resources` in the IG. If it does, then the code is not working.

```
Resource: FormTemplate
Parent: Resource
Id: formtemplate
Title: "FHIR2 Form Template"
Description: "FHIR2 Form Template"
* template 1..1 BackboneElement "The root of the doc"
  * templateStatus 1..1 string "the template status"

Instance: FHIR2FormTemplateExample
InstanceOf: FormTemplate
Title: "Example : Questionnaire"
Description: "Example : Questionnaire"
Usage: #example
* template.templateStatus = "draft"
```
Also try to test if a normal Instance (like a Patient Instance) still displays under `resources` in IG.

Thanks for reviewing! 💯
